csaf_2.1/json_schema/csaf_json_schema.json

```diff
@@ -1288,6 +1288,36 @@
                     "cvss_v4": {
                       "$ref": "https://www.first.org/cvss/cvss-v4.0.json"
                     },
+                    "epss": {
+                      "title": "EPSS",
+                      "description": "Contains the EPSS data.",
+                      "type": "object",
+                      "required": [
+                        "percentile",
+                        "probability",
+                        "timestamp"
+                      ],
+                      "properties": {
+                        "percentile": {
+                          "title": "Percentile",
+                          "description": "Contains the rank ordering of probabilities from highest to lowest.",
+                          "type": "string",
+                          "pattern": "^(([0]\\.([0-9])+)|([1]\\.[0]+))$"
```

Comment:
Maybe we are better off using a number here and adding test for that number to be in the expected range and display form?

Reply:
I'm +-0 about that idea. The official API at https://api.first.org/epss/ has the response as string. So it might be the least surprise, if we stick with the strings. However, from the JSON schema point of view, the `number` really makes sense. We wouldn't need to do an extra test - we can use `minimum` and `maximum`.